docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
+1. Enter the name of the module you would like to add to your robot.
+   To find the name of a module you're interested in, you can:
+
+   - Start typing to search for modules by name.
+     Modules available from the Viam Registry will be listed under the `From Registry` section of the search results.
+   - [Browse the Viam Registry](https://app.viam.com/modules) directly to search available modules.
+
+   {{<imgproc src="extend/modular-resources/configure/add-module-from-registry.png" resize="400x" declaredimensions=true alt="The add a component modal showing results for the intel realsense module ">}}
+
+1. When you have entered the name of the module that you would like to add to your robot, select the matching module in the search results and click the **Add module** button.
+
+When you add a module from the Viam Registry, the custom modular component it provides appears under the **Components** subtab like any other component.
+You can also [configure the module itself](#configure-a-module-from-the-viam-registry) from its configuration pane in the **Modules** subtab.
+
+{{<imgproc src="extend/modular-resources/configure/conf-component-from-module.png" resize="400x" declaredimensions=true alt="The components subtab of the config tab showing the camera component configuration pane for the realsense module">}}
+
+If the module requires that specific **Atrributes** be configured, click the **URL** link in the module's configuration pane to view the specific documentation for that module on the module's Github page.
+
+To delete a module added from the Viam Registry, click the trash can icon in the upper-right corner of the module configuration pane in the **Components** tab.
+
+### Configure a module from the Viam Registry
+
+Once you have added a module from the Viam Registry, you can view and configure the module from the **Modules** subtab:
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Modules** subtab. All modules you have added to your robot appear under the **Deployed** section.
+
+This pane lists the models provided by the module, any [components](/components/) on your robot that are currently using those models, and allows you to configure [how the module updates](#configure-version-update-management-for-a-registry-module) when a new version is available from the Viam Registry.
+
+{{<imgproc src="extend/modular-resources/configure/conf-module-from-registry.png" resize="1000x" declaredimensions=true alt="The module subtab of the config tab showing the realsense custom module configuration pane includes the update management section showing version update management options version type, set to Patch (X.Y.Z) and version set to 0.0.3">}}
+
+#### Configure version update management for a Registry module
+
+When you add and configure a module on your robot, the module is automatically set to pin to the specific patch release version you added.
```

Comment:
[opt] "automatically set to pin to" sounds potentially hard to parse for some users but I don't have a better idea that's still succinct. But also, isn't "add and configure" redundant?
```suggestion
When you configure a module on your robot, by default it will pin to the specific patch release version you added.
```